src/reanimated2/platform-specific/webAnimations.ts

```diff
@@ -0,0 +1,278 @@
+import { Animations, AnimationsData, WebEasings } from './webAnimationsData';
+import type { AnimationTypes } from './webAnimationsData';
+import { parseAnimationObjectToKeyframe } from './animationParser';
+import type { TransformProperties } from './animationParser';
+import type { TransformsStyle } from 'react-native';
+import type {
+  BaseAnimationBuilder,
+  EntryExitAnimationFunction,
+  ILayoutAnimationBuilder,
+} from '../layoutReanimation';
+
+export const WEB_ANIMATIONS_ID = 'webAnimationsStyle';
+
+export interface AnimationConfig {
+  animationName: string;
+  duration: number;
+  delay: number;
+  easing: string;
+  callback: (() => void) | null;
+}
+
+interface CustomConfig {
+  easingV?: () => number;
+  durationV?: number;
+  delayV?: number;
+  randomizeDelay?: boolean;
+  callbackV?: () => void;
+}
+
+type ConfigType =
+  | BaseAnimationBuilder
+  | ILayoutAnimationBuilder
+  | typeof BaseAnimationBuilder
+  | EntryExitAnimationFunction
+  | Keyframe
+  | CustomConfig;
+
+/**
+ *  Creates `HTMLStyleElement`, inserts it into DOM and then inserts CSS rules into the stylesheet.
+ *  If style element already exists, nothing happens.
+ */
+export function insertWebAnimations(): void {
```

Comment:
```suggestion
export function insertWebAnimations() {
```

Reply:
Done in [7f8735d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/7f8735deb90c38024a96cbe8a49260758ee8097f)